py/selenium/webdriver/phantomjs/service.py

```diff
@@ -75,13 +75,15 @@ def start(self):
                                             stdout=self._log, stderr=self._log)
 
         except Exception as e:
-            raise WebDriverException("Unable to start phantomjs with ghostdriver.", e)
+            raise WebDriverException(
```

Comment:
it was originally put in there to handle stacktraces that would be returned by the remote end (the browser drivers themselves) and then put into the python Exception.

This is essentially piggy-backing on that 'feature' of the WebDriverException class.